packages/storybook/src/stories/icons/SubtractIcon.stories.tsx

```diff
@@ -0,0 +1,42 @@
+import SubtractIcon from '@littlespoon/icons/src/SubtractIcon'
+import { ComponentMeta, ComponentStory } from '@storybook/react'
+
+export default {
+  title: 'Design System/Icons/SubtractIcon',
+  component: SubtractIcon,
+} as ComponentMeta<typeof SubtractIcon>
+
+const Template: ComponentStory<typeof SubtractIcon> = (args) => <SubtractIcon {...args} />
+
+export const Default = Template.bind({})
+Default.args = {}
+
+export const ExtraSmall = Template.bind({})
+ExtraSmall.args = {
+  size: 'xsmall',
+}
+
+export const Small = Template.bind({})
+Small.args = {
+  size: 'small',
+}
+
+export const Medium = Template.bind({})
+Medium.args = {
+  size: 'medium',
+}
+
+export const Large = Template.bind({})
+Large.args = {
+  size: 'large',
+}
+
+export const Fill = Template.bind({})
+Fill.args = {
+  fill: 'skyblue',
+}
```

Comment:
Does the fill look a little uncentered? https://deploy-preview-589--littlespoon.netlify.app/?path=/story/design-system-icons-subtracticon--fill

![Screen Shot 2022-08-24 at 5 21 37 PM](https://user-images.githubusercontent.com/10594555/186526087-dbd67862-d69c-42a7-9644-3575969970c4.png)

Reply:
I think the svg file itself has some alignment issue.
I updated the svg file and pushed a new commit
![Screen Shot 2022-08-25 at 12 04 28 PM](https://user-images.githubusercontent.com/93789729/186714628-fb51bce5-34de-4c51-ba79-105046b4a76c.png)